### ISSUE TITLE
Integration with Absinthe GraphQL

### ISSUE THREAD

**Issue by Yamilquery** [slashdotdash/conduit#13]
I'm really interested to apply the CQRS approach in some of our future projects.
I've watching this awesome repo and I really like so much. But I'd wish have an example of how can we implement this using something like absinthe to get all the power of GraphQL with the CQRS approach.

I really want to know the vision from the experts about combine this technology with the CQRS pattern.

Thank you!

**Comment by slashdotdash** [comment]
@Yamilquery Yes I plan to either include a chapter on using GraphQL by integrating Commanded with Absinthe. I'm using this combination in a production app with commands defined as GraphQL mutations and subscription being updating via the [`after_update/3` callback in Commanded Ecto projections](https://github.com/commanded/commanded-ecto-projections#after_update-callback).
